app/models/comment.rb

```diff
@@ -1,2 +1,4 @@
 class Comment < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id', inverse_of: :comments
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id', inverse_of: :comments
 end
```

Comment:
- [ ] [REQUIRED] Let's also ensure that the comment model has a method that updates the comments counter for a post.